### PR TITLE
fix(athena-driver): column order swapping for Athena

### PR DESCRIPTION
**Pull request by zinychz**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: packages/cubejs-athena-driver/src/AthenaDriver.ts [hunk 1]
```diff
@@ -63,6 +63,24 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
...
+    const [schema, name] = table.split('.');
+
+    const columns = (await this.query(
+      `SELECT columns.column_name as ${this.quoteIdentifier('column_name')},
+             columns.table_name as ${this.quoteIdentifier('table_name')},
+             columns.table_schema as ${this.quoteIdentifier('table_schema')},
+             columns.data_type  as ${this.quoteIdentifier('data_type')}
+      FROM information_schema.columns
+      WHERE table_name = ${this.param(0)} AND table_schema = ${this.param(1)}
+      ORDER BY 1, 2, 3, 4`,
```
Comment (cristipp) [inline packages/cubejs-athena-driver/src/AthenaDriver.ts:76]: Nit: `ORDER BY 1` suffices, the other columns are redundant.

File: packages/cubejs-athena-driver/src/AthenaDriver.ts [hunk 2]
```diff
@@ -63,6 +63,24 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
     this.athena = new Athena(this.config);
   }
 
+  public async tableColumnTypes(table: string) {
+    const [schema, name] = table.split('.');
```
Comment (cristipp) [inline packages/cubejs-athena-driver/src/AthenaDriver.ts:67]: Nit: `table.split('.', 2)` so we are sure there are no surprises.

File: packages/cubejs-athena-driver/src/AthenaDriver.ts [hunk 3]
```diff
@@ -63,6 +63,24 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
     this.athena = new Athena(this.config);
   }
 
+  public async tableColumnTypes(table: string) {
```
Comment (cristipp) [inline packages/cubejs-athena-driver/src/AthenaDriver.ts:66]: Not 100% why/where the order of columns returned by the source DB matters. Shouldn't Cube column order be defined by the order of measures/dimensions/timeDimensions? Can you please add a small jsdoc comment line to explain /why/ the `ORDER BY columns.column_name` clause is necessary?

**Comment by cristipp** [comment]
If I understand correctly, this PR stabilizes the column order. The fix is done only for AthenaDriver. Are other drivers potentially subject to the same issue?

File: packages/cubejs-athena-driver/src/AthenaDriver.ts (hunk 1 above)
Comment (zinychz) [reply packages/cubejs-athena-driver/src/AthenaDriver.ts:76]: You are right, agree

**Comment by cristipp** [comment]
Generally speaking, a bug fix should have a test associated with it, which should fail without the fix and pass with the fix.

**Pull request closed without merging**